### DISCUSSION TITLE
How many bots can this computer handle?

### DISCUSSION THREAD

**Discussion by xenfyer** [KevinBytesTheDust/CVAmp#273] [Q&A] [answered]
Hello! How many active windows can this computer handle?
CPU: Intel Core i7-12700K
RAM: 128 GB

I'm trying to figure out what kind of computer I'll need for my purposes. Thank you!

**Comment by jlplenio** [comment, marked as answer]
If the stream offers 160p, then I would estimate around 150 instances.
